compiler/ballerina-lang/src/main/java/io/ballerina/projects/environment/ModuleLoadRequest.java

```diff
@@ -86,6 +90,10 @@ public boolean injected() {
         return dependencyResolvedType == DependencyResolutionType.INJECTED;
     }
 
+    public Optional<Location> location() {
```

Comment:
This is also not used. Can we remove this location?

Reply:
Removed